src/lib/viewers/doc/DocControls.tsx

```diff
@@ -7,6 +7,7 @@ import FullscreenToggle, { Props as FullscreenToggleProps } from '../controls/fu
 import PageControls, { Props as PageControlsProps } from '../controls/page';
 import ThumbnailsToggle, { Props as ThumbnailsToggleProps } from '../controls/sidebar';
 import ZoomControls, { Props as ZoomControlsProps } from '../controls/zoom';
+import { AnnotationMode } from '../../AnnotationControls';
```

Comment:
We already import AnnotationControls above.

Reply:
I think we have `AnnotationsControls`, not `AnnotationControls`.

Reply:
Yes, that is intentional. We intend to remove the latter component soon.